playbooks/toolbox-mock-images.yaml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Set variables
+  set_fact:
+    toolbox_dirs:
+      "{{ toolbox_dirs }} + [ '{{ zuul.project.src_dir  }}/fedora-toolbox-{{ item }}' ]"
+    toolbox_images:
+      "{{ toolbox_images }} + [ '{{ registry_url }}/f{{ item }}/fedora-toolbox:{{ item }}' ]"
+  loop: "{{ fedora_versions }}"
+
+- name: Ensure the toolbox images are been pulled to the local registry
+  command: "podman pull {{ item }}"
+  register: _podman
+  until: _podman.rc == 0
+  retries: 5
+  delay: 10
+  loop: "{{ toolbox_images }}"
+
+- name: Ensure the toolbox image is at local directory
+  command:
+    cmd: "skopeo copy containers-storage:{{ item.0 }} dir:{{ item.1 }}"
+    creates: "{{ item.1 }}/manifest.json"
+  register: _skopeo
+  loop: "{{ toolbox_images | zip(toolbox_dirs) | list }}"
```

Comment:
`list` is a filter:
https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#combining-items-from-multiple-lists-zip-and-zip-longest

Reply:
I see. Thanks for the link, @juanje !

In the end, I had to [open-code the loops](https://github.com/containers/toolbox/pull/704) because all the other images, except the one for Fedora 29, have moved to a newer path on `registry.fedoraproject.org`, and I couldn't trust my Ansible skills to properly parametrize that.

We could either wait for a few more months when Fedora 32 gets EOLed, and we can use the F32 image instead of the F29 one and all the URLs will be in the same pattern again. Or we could use your help to improve things now in a separate PR. :)